pkg/controllers/user/nodesyncer/nodessyncer.go

```diff
@@ -490,7 +482,8 @@ func statusEqualTest(proposed, existing corev1.NodeStatus) bool {
 	return true
 }
 
-func (m *NodesSyncer) convertNodeToNode(node *corev1.Node, existing *v3.Node, pods map[string][]*corev1.Pod) (*v3.Node, error) {
+// convertNodeToMachine takes a core.v1 Node, a possibly existing machine, and the pods running on the core.v1 Node
```

Comment:
Comment should give some context on the function if needed. The sig already tells you what goes into it